core/tests/test_utils.py

```diff
@@ -247,6 +250,8 @@ def mock_load_template(filename: str) -> str:
     Args:
         filename: str. The name of the file for which template is to be
             returned.
+        template_is_aot_compiled: bool or None. False by default. Use
```

Comment:
What does None mean? This should be explained in the docstring (though it would be much better to have it just be bool throughout if possible).

Reply:
My understanding is that you can either pass a boolean or pass nothing. If you pass nothing, false will be the value assigned to template_is_aot_compiled.